BParkingNano/python/BToKLL_cff.py

```diff
@@ -23,12 +23,13 @@
     beamSpot = cms.InputTag("offlineBeamSpot"),
     kaonSelection = cms.string(''),
     preVtxSelection = cms.string(
-        'pt > 3. && userFloat("min_dr") > 0.01 '
-        '&& mass < 6 && mass > 4.5'
+        'pt > 3. && userFloat("min_dr") > 0.03 '
+        '&& mass < 7. && mass > 3.'
```

Comment:
Bump the mass to > 4 GeV

Reply:
why mindr>0.03 ? also do we really need the extended M(B) range? the further we are from B, more different thr bkg composition will be